packages/p2p/src/components/buy-sell/buy-sell-header.jsx

```diff
@@ -98,12 +98,20 @@ const BuySellHeader = ({ table_type }) => {
                         placeholder={isDesktop() ? localize('Search by nickname') : localize('Search')}
                     />
                     <SortDropdown />
-                    <Icon
-                        className='buy-sell__header-row--filter'
-                        icon='IcFilter'
-                        onClick={() => general_store.showModal({ key: 'FilterModal', props: {} })}
-                        size={40}
-                    />
+                    <div style={{ position: 'relative' }} className='buy-sell__header-row--filter'>
```

Comment:
can we move this inline style to class?

Reply:
I forgot to remove this and add that in the class, thanks for spotting it 😅